pkg/controller/tidbcluster/tidb_cluster_control.go

```diff
@@ -281,6 +293,14 @@ func (c *defaultTidbClusterControl) updateTidbCluster(tc *v1alpha1.TidbCluster)
 		}
 	}
 
+	// Replace volumes if necessary. Note: if enabled, takes precedence over pvcModifier.
+	if features.DefaultFeatureGate.Enabled(features.VolumeReplacing) {
```

Comment:
After https://github.com/pingcap/tidb-operator/commit/df734da0eb0c73d71f09944d8f31d604631d4bfa I think resizing is turned on by default anyways.

I've tested after this commit, and replacer is designed to take precedence over modification if enabled. Basically the Sync here will keep requeuing until all replacement is complete, so it will never give pvcModifier below a chance. When replacement is done, modifier will have nothing to do. Without precedence pvc modifier may keep failing for changes it can't handle and never let replacer run.

This means if a modification would have been enough, replacement feature will still replace instead of resizing, this is a downside to note. Maybe in future we can coordinate if pvc modifier can handle it let it do instead of replace, but didn't complicate it in this PR.

So it should be fine to turn on both, but only one will take effect. Didn't want to throw an error in case we default modification feature later. Let me know what you think.